qute/utilities/styling.py

```diff
@@ -166,19 +168,26 @@ def _getAvailableStyles():
     """
     styles = dict()
 
+    # -- Only check for qss or css files
+    valid_extensions = {'.css', '.qss'}
+
     for location in constants.QUTE_STYLE_LOCATIONS:
 
         # -- Skip it if it is not accessible
-        if not os.path.exists(location):
+        if not os.path.isdir(location):
```

Comment:
Accounts for the possibility that we were given a filepath (os.path.exists would have allowed).
This being said, it might be useful to allow .zip files?